behave/features/steps/common_steps.py

```diff
@@ -46,6 +47,22 @@ def step_impl(context, hyperlink_text):
     context.helperfunc.click_button(By.LINK_TEXT, hyperlink_text)
 
 
+@step(u'I select the \'Sign out\' link')
+def step_impl(context):
+    # We need to be on a page that we can control, otherwise the current page could have a modal dialog visible which
+    # could impact how we can interact with menu elements
+    context.helperfunc.open(f"{CLA_FRONTEND_URL}")
+    page = context.helperfunc
+    context.header = context.helperfunc.find_by_xpath("//header[@id='global-header']")
+    # find the menu link
+    user_menu = "//div[@class='UserMenu']/a"
+    # wait for the menu link to be visible then click
+    WebDriverWait(page.driver(), 10).until(EC.element_to_be_clickable((By.XPATH, user_menu))).click()
```

Comment:
Can we add this wait to be clickable into the click_button functionality on line 91 of `helper_base.py`?

Reply:
Good point, updated